src/main.cpp

```diff
@@ -810,7 +810,7 @@ void EraseOrphansByTime() EXCLUSIVE_LOCKS_REQUIRED(cs_orphancache)
     // every time a tx enters the mempool but just once every 5 minutes is good enough.
     if (GetTime() <  nLastOrphanCheck + 5*60)
         return;
-    int64_t nOrphanTxCutoffTime = GetTime() - GetArg("-mempoolexpiry", DEFAULT_MEMPOOL_EXPIRY) * 60 * 60;
+    int64_t nOrphanTxCutoffTime = GetTime() - GetArg("-orphanpoolexpiry", DEFAULT_ORPHANPOOL_EXPIRY) * 60 * 60;
```

Comment:
If you use a CTweak for this parameter, you'll automatically be able to change it using bitcoin-cli (and command line and conf file access).  Will people want to tweak their orphan pool expiry without restarting bitcoind?